src/pre_defined_classes/mathematical_program/medium/q3/MainTest.java

```diff
@@ -0,0 +1,42 @@
+package pre_defined_classes.mathematical_program.medium.q3;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.DoubleLiteral;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons
+    final double cos2 = 0.25;
+    final double sin2 = 0.75;
+    final double sum = 1.0;
```

Comment:
Again, why put numbers you only use in the math test up here? Other than that it's good.